### ISSUE TITLE
Is there a polling alternative?

### ISSUE THREAD

**Issue by EdwardLyle** [rqlite/rqlite#541]
Yet all clients have to poll the server for database changes. In my new project a few hundred clients have to poll the rqlite server every second. It would be great when we can subscribe to the server like in PubSubSQL. Is this possible with rqlite?

**Comment by otoolep** [comment]
No, rqlite doesn't support anything like that right now. You mean something like:

https://github.com/rqlite/rqlite/issues/127

**Comment by otoolep** [comment]
https://philipotoole.com/rqlite-9-0-real-time-change-data-capture-for-distributed-sqlite/
